omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/OmniLocalExecutionPlanner.java

```diff
@@ -468,27 +467,18 @@ public LocalExecutionPlan plan(TaskContext taskContext, StageExecutionDescriptor
             return null;
         }
 
-        // if there is a data type not support by OmniRuntime, then fall back
```

Comment:
I suggest we should keep this code block for two reasons:
1. We have a expliciit position to check type support for each operator.
2. No need to modify in OperatorFactories's constructor